cli/build-public-library.js

```diff
@@ -107,7 +121,8 @@ function writeTSConfig() {
     ),
     compilerOptions: {
       lib: ['dom', 'es6']
-    }
+    },
+    angularCompilerOptions: {},
```

Comment:
I'd rather the generated `tsconfig.json` remained as it is today by default (no `angularCompilerOptions` object created if empty), instead of adding an empty `angularCompilerOptions` object, just to be careful about introducing any potential problems for existing consumers.

Reply:
Reverted
- This was added so I'd be sure the `angularCompilerOptions` was defined before attempting to set properties while merging.
- The property merging now functionality will be smart and create the `angularCompilerOptions/compilerOptions` (if needed) if there are any flags to merge.